packages/core/src/browser/style/tree.css

```diff
@@ -73,6 +73,6 @@
 }
 
 .theia-TreeNodeSegment mark {
-    background-color: var(--theia-highlight-background-color);
-    color: var(--theia-highlight-color);
+    background-color: var(--theia-search-match-color1);
```

Comment:
That style is defined in search-in-workspace extension (https://github.com/theia-ide/theia/blob/master/packages/search-in-workspace/src/browser/styles/index.css#L10). 
If we also use this in core we should move it to core.